### DISCUSSION TITLE
Why not use eprintln! for error handling?

### DISCUSSION THREAD

**Discussion by ghost** [tokio-rs/tokio#5393] [General]
E.g., https://docs.rs/tokio/latest/tokio/net/struct.TcpListener.html The error handling uses println!("couldn't get client: {:?}", e) , but it will be printed to standard output. Eprintln prints to standard error.

**Comment by Darksonn** [comment]
It's just an example. There isn't any reason.

**Comment by carllerche** [comment]
Feel free to change it, though. I think a PR to switch to `eprintln` is reasonable personally.
